### ISSUE TITLE
DataStore_Reputations.lua attempt to index local 'info' (a nil value)

### ISSUE THREAD

**Issue by JamesMDenton** [Thaoky/Altoholic_Retail#36]
```
 ...dOns/DataStore_Reputations/DataStore_Reputations.lua:35: attempt to index local 'info' (a nil value)
[string "@DataStore_Reputations/DataStore_Reputations.lua"]:35: in function <...dOns/DataStore_Reputations/DataStore_Reputations.lua:33>
[string "@DataStore_Reputations/DataStore_Reputations.lua"]:298: in function <...dOns/DataStore_Reputations/DataStore_Reputations.lua:294>
[string "@DataStore_Reputations/DataStore_Reputations.lua"]:437: in function <...dOns/DataStore_Reputations/DataStore_Reputations.lua:436>
[string "@DataStore_Reputations/DataStore_Reputations.lua"]:521: in function `callback'
[string "@DataStore/Events/Addon.lua"]:17: in function <DataStore/Events/Addon.lua:13>

Locals:
index = 7
info = nil
(*temporary) = nil
(*temporary) = 7
(*temporary) = nil
(*temporary) = nil
(*temporary) = nil
(*temporary) = nil
(*temporary) = "attempt to index local 'info' (a nil value)"
C_Reputation = <table> {
}
```

Pretty constantly being spammed with this as I'm questing through the campaign in the Dragon Isles.

**Comment by Thaoky** [comment]
Should be fixed in 11.0.002, please reopen if it is not.

**Comment by Odysseas68** [comment]
I have update to v11.0.002 and i got the same error:

1x ...dOns/DataStore_Reputations/DataStore_Reputations.lua:35: attempt to index local 'info' (a nil value)
[string "@DataStore_Reputations/DataStore_Reputations.lua"]:35: in function <...dOns/DataStore_Reputations/DataStore_Reputations.lua:33>
[string "@DataStore_Reputations/DataStore_Reputations.lua"]:298: in function <...dOns/DataStore_Reputations/DataStore_Reputations.lua:294>
[string "@DataStore_Reputations/DataStore_Reputations.lua"]:437: in function <...dOns/DataStore_Reputations/DataStore_Reputations.lua:436>
[string "@DataStore_Reputations/DataStore_Reputations.lua"]:521: in function `callback'
[string "@DataStore/Events/Addon.lua"]:17: in function <DataStore/Events/Addon.lua:13>

Locals:
index = 4
info = nil
(*temporary) = nil
(*temporary) = 4
(*temporary) = nil
(*temporary) = nil
(*temporary) = nil
(*temporary) = nil
(*temporary) = "attempt to index local 'info' (a nil value)"
C_Reputation = <table> {
}
